common/libs/values.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import type Big from 'big.js';
+
+export function padLeft(n: string, width: number, z: string = '0'): string {
+  return n.length >= width ? n : new Array(width - n.length + 1).join(z) + n;
+}
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
+  let reminder = n;
+  let result = [];
+  while (!reminder.eq(0)) {
+    const mod = reminder.mod(16);
+    reminder = reminder.div(16).round(0, 0);
```

Comment:
It is definitely used, because we are in loop. Assignment by value vs by reference wouldnt make a difference in any (most?) languages here, because it is the same variable inside same function.